front/app/containers/Admin/projects/project/permissions/components/ModeratorListRow.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Text, Box } from '@citizenlab/cl2-component-library';
+import Avatar from 'components/Avatar';
+import Button from 'components/UI/Button';
+import { Row } from 'components/admin/ResourceList';
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+import { isAdmin } from 'services/permissions/roles';
+import { IUserData } from 'services/users';
+import styled from 'styled-components';
+import { deleteProjectModerator } from 'services/projectModerators';
+
+interface Props {
+  isLastItem: boolean;
+  moderator: IUserData;
+  projectId: string;
+}
+
+const PendingInvitation = styled.span`
+  font-style: italic;
+`;
+
+const UnknownName = styled.span`
+  font-style: italic;
+`;
+
+const ModeratorListRow = ({ isLastItem, moderator, projectId }: Props) => {
+  const { formatMessage } = useIntl();
+  const moderatorId = moderator.id;
+  const firstName = moderator.attributes.first_name;
+  const lastName = moderator.attributes.last_name;
+  const invitationPending = moderator.attributes.invite_status === 'pending';
+  const displayName = invitationPending ? (
```

Comment:
In this case, it might be better to just use the if statements and `let`. The nested ternary can be hard to read

Reply:
👍 I have simplified it. Unknown name should not be needed. We always have at least a first name if the user is registered.